### DISCUSSION TITLE
Farmer Public Key (pk) List Location

### DISCUSSION THREAD

**Discussion by g1thub11** [Chia-Network/chia-blockchain#7711] [Q&A] [answered]
Where can I find the farmer's pk list (running 1.2.2 on Windows)?

I know about "chia keys show". I am asking about the location of the file where the pk list is stored.

**Comment by kanna5** [comment, marked as answer]
Farmer key is derived from master key, master key is stored in python `keyring`. Actual location of the file is platform dependent. On Windows it is managed by `Credential Manager`.
